Integrations/integration-PassiveTotal.yml

```diff
@@ -484,6 +489,60 @@ script:
     - contextPath: Domain(val.Malicious)
       description: Malicious Domains
     description: Get malware data
+  - name: url
+    arguments:
+    - name: url
+      required: true
+      default: true
+      description: The URL to get malware for
+    - name: samples
+      description: The number of malware samples to return
+      defaultValue: "10"
+    - name: threshold
+      description: Above this number of samples we will consider this to be malicious
+      defaultValue: "10"
+    outputs:
+    - contextPath: IP.Address
+      description: Bad IPs found during enrichment
+    - contextPath: IP.Malicious.Vendor
+      description: For malicious IPs, the vendor that made the decision
+    - contextPath: IP.Malicious.Description
+      description: For malicious IPs, the reason for the vendor to make the decision
+    - contextPath: Domain.Name
+      description: Bad domains found during enrichment
+    - contextPath: Domain.Malicious.Vendor
+      description: For malicious domains, the vendor that made the decision
+    - contextPath: Domain.Malicious.Description
+      description: For malicious domains, the reason for the vendor to make the decision
+    - contextPath: File.MD5
+      description: Bad hash MD5
+    - contextPath: File.SHA1
+      description: Bad hash SHA1
+    - contextPath: File.SHA256
+      description: Bad hash SHA256
+    - contextPath: File.Malicious.Vendor
+      description: For malicious files, the vendor that made the decision
+    - contextPath: File.Malicious.Description
+      description: For malicious files, the reason for the vendor to make the decision
+    - contextPath: DBotScore.Indicator
+      description: The indicator we tested
+    - contextPath: DBotScore.Type
+      description: The type of the indicator
+    - contextPath: DBotScore.Vendor
+      description: Vendor used to calculate the score
+    - contextPath: DBotScore.Score
+      description: The actual score
+    important:
+    - contextPath: File(val.Malicious)
```

Comment:
No it will not. When running with a URL, pt-malware only adds relevant context entries. 
This is how pt-malware is designed, so I just duplicated it